tickets/src/middlewares/eventMiddleware.js

```diff
@@ -71,14 +72,37 @@ const eventMiddleware = config => {
         ) {
           eventService
             .saveEvent(action.data.message.event, action.signature)
-            .catch(error => dispatch(eventError(error)))
+            .then(() => dispatch(savedEvent(action.data.message.event)))
+            .catch(error => {
+              dispatch(eventError(error))
+            })
         }
 
         // Load an event from locksmith
         if (action.type === LOAD_EVENT) {
-          eventService.getEvent(action.address).then(event => {
-            dispatch(updateEvent(event))
-          })
+          eventService
+            .getEvent(action.address)
+            .then(event => {
+              dispatch(updateEvent(event))
+            })
+            .catch(() => {
+              // Not Found?
+              dispatch(
```

Comment:
In case of 404 we actually create a "placeholder" event which can then be displayed and saved.

Reply:
👍